tests/tests.wake

```diff
@@ -82,59 +83,45 @@ export def runTests (cmdline: List String): Result String Error =
             def failed = len failures
             def total = len list
             def pass = total - failed
-            "  - {category}: {str pass}/{str total} tests pass",
-            failures
+            "  - {category}: {str pass}/{str total} tests pass", failures
         catWith "\n" text
 
     def report =
-        results
-        | groupBy (_.getTripleFirst <* _.getTripleFirst)
-        | map formatCategory
-        | catWith "\n"
+        results | groupBy (_.getTripleFirst <* _.getTripleFirst) | map formatCategory | catWith "\n"
 
     def _ = printlnLevel logWarning "{wakeDir}/wake unit testing results:\n{report}\n---"
 
     require Pass _ = findFailFn getTripleThird results
 
     Pass "All {len tests | str} tests completed successfully."
 
+
 export def runUnitTests _: Result Unit Error =
-    def testDirectory = "{here}/wake-unit"
-    require Pass testScript =
-      source "{testDirectory}/unit-test.sh"
+    def testDirectory = "{@here}/wake-unit"
+    require Pass testScript = source "{testDirectory}/unit-test.sh"
 
-    def testName =
-        testDirectory
-        | replace `^${quote here}/` ""
+    def testName = testDirectory | replace `^${quote @here}/` ""
 
     def readFile list = match list
         Some file =
             require Pass content = read file
             Pass (Some content)
-        None =
-            Pass None
+        None = Pass None
 
-    require Pass expectedStdout =
-        source "{testDirectory}/stdout"
-        | getPass
-        | readFile
+    require Pass expectedStdout = source "{testDirectory}/stdout" | getPass | readFile
 
-    require Pass expectedStderr =
-        source "{testDirectory}/stderr"
-        | getPass
-        | readFile
+    require Pass expectedStderr = source "{testDirectory}/stderr" | getPass | readFile
 
-    require Pass visibleFiles =
-        sources testDirectory `.*`
+    require Pass visibleFiles = sources testDirectory `.*`
 
-    def inTestDir path =
-        relative testDirectory path
+    def inTestDir path = relative testDirectory path
 
-    require Pass wakeUnitBin =
-        wakeUnitToTest Unit
+    require Pass wakeUnitBin = wakeUnitToTest Unit
 
     def testJob =
-        makeExecPlan ("./{testScript.getPathName.inTestDir}", wakeUnitBin.getPathName.inTestDir, Nil) (wakeUnitBin, visibleFiles)
+        makeExecPlan
+        ("./{testScript.getPathName.inTestDir}", wakeUnitBin.getPathName.inTestDir, Nil)
+        (wakeUnitBin, visibleFiles)
         | setPlanDirectory testDirectory
         | setPlanLabel "testing: {testName}"
         | setPlanStdout logNever
```

Comment:
I can certainly see how this would be *legal* but it's unclear enough that it's probably a good PR comment to ask people to put the command on a separate line at this point.